packages/babel/src/api/browser.js

```diff
@@ -57,6 +57,14 @@ transform.load = function (url, callback, opts = {}, hold) {
 };
 
 /**
+ * <script> tags may be children of documents other than window.document,
+ * e.g. inside imported documents via <link rel="import" ...>.
+ * currentDocument points to the document to which this script is attached.
+ */
+var currentScript = global.document._currentScript || global.document.currentScript;
```

Comment:
> If you are using webcomponents.js, use document._currentScript instead of document.currentScript. The underscore is used to polyfill the currentScript property which is not available in all browsers.

http://webcomponents.org/articles/introduction-to-html-imports/

I couldn't get tests work with `var currentScript = global.document.currentScript || global.document._currentScript ;` so I reversed it.